scapy/utils.py

```diff
@@ -2567,6 +2567,25 @@ def whois(ip_address):
             break
     return b"\n".join(lines[3:])
 
+
+class ThreadSafeInt(object):
```

Comment:
This is again a leftover of my atomic assumption. Anyway I would need to pass a object instead of an int from the FloodGenerator to sendrcv. Do you know an elegant way to pack an int into an object to be able to pass it by reference?